pkg/kubelet/cm/devicemanager/plugin/v1beta1/stub.go

```diff
@@ -76,20 +81,36 @@ func defaultAllocFunc(r *pluginapi.AllocateRequest, devs map[string]pluginapi.De
 	return &response, nil
 }
 
+// stubRegisterControlFunc is the function called when a registration request is received from Kubelet
+type stubRegisterControlFunc func() bool
```

Comment:
so this Stub, is it only for the sample plugin? Or we expect more prod usage?

Reply:
`stubRegisterControlFunc` is referenced in the `Stub` struct above which can be used by other device plugins for their bootstapping with the help of `NewDevicePluginStub` initializer. Sample device plugin happens to be such implementation that we use for testing.